holidays/locale/en_IN/LC_MESSAGES/XNSE.po

```diff
@@ -0,0 +1,97 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+# SOME DESCRIPTIVE TITLE
+# This file is distributed under the same license as the Holidays package.
+# FIRST AUTHOR saheelsapovadia23@gmail.com, 2025.
+msgid ""
+msgstr ""
+"Project-Id-Version: Holidays 0.79\n"
+"POT-Creation-Date: 2025-08-20 20:21+0000\n"
+"PO-Revision-Date: 2025-08-20 20:21+0000\n"
+"Last-Translator: Saheel Sapovadia saheelsapovadia23@gmail.com\n"
+"Language-Team: Holidays Localization Team\n"
+"Language: en_IN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Lingva 5.0.6\n"
+"X-Source-Language: en_IN\n"
+
+msgid "Republic Day"
+msgstr "Republic Day"
```

Comment:
For `default_language` (in this case, `en_IN`), all `msgstr` should be left empty
```suggestion
msgid "Republic Day"
msgstr ""
```